knowledge-base/fiddler-archives.md

```diff
@@ -0,0 +1,131 @@
+        ```
+
+    * **sessid#_s.txt**&mdash;The file contains the raw server HTTP(S) response.
+
+        _An example for raw server response file_
+        ```xml
+        HTTP/1.1 502 Bad Gateway
+        Content-Length: 82269
+        Connection: close
+        Cache-Control: max-age=0
+        Content-Type: text/html; charset=UTF-8
+
+        <!DOCTYPE html>
+        <html lang="en">
+
+        <head>
+            <meta charset="UTF-8">
+            <meta http-equiv="X-UA-Compatible" content="IE=edge">
+            <meta name="viewport" content="width=device-width, initial-scale=1.0">
+            <title>Certificate error</title>
+        </head>
+
+        <body>
+            <fdl-card>
+                <div class="card__background-image warning-icon"></div>
+                <div class="card__background-image"></div>
+            </fdl-card>
+        </body>
+        </html>
+        ```
+
+    * **sessid#_m.xml**&mdash;The file contains metadata, including session flags (element `SessionFlags`), certificate chain information (element `CertificateChainInfo`), socket reuse information, etc.
+
+    * **sessid#_w.txt**&mdash;(Optional) The file contains WebSocket messages.
+
+    * **sessid#_g.txt**&mdash;(Optional) The file contains GRPC messages.
+
+* **[Content_Types.xml]**&mdash;A metadata file that specifies a few MIME types, so the archive is compatible with `System.IO.Packaging` or other clients that support the Open Packaging Conventions.
+
+## Using SAZ with Fiddler Everywhere
+
+The Fiddler Everywhere uses SAZ behind the scenes for each session explicitly saved locally or in the cloud. The application also provides options to manually export and import captured and saved traffic through the UI. 
+
+>tip You can set the Fiddler Everywhere application as the default application for opening SAZ files through the operating system's settings (for example, on Windows 11 through **Settings > Apps > Default Apps**).
+
+ SAZ files use the following UI options:
+
+- Use the **Export** option from the Live Traffic context menu and then choose **Fiddler Archive (SAZ)** as format.
+
+    ![Export SAZ from Live Traffic](../images/kb/saz/saz-export-live-traffic.png)
+
+- Use the **Export** option from the **Sessions** list's context menu and then choose **Fiddler Archive (SAZ)** as format.
+
+    ![Export SAZ from Lthe saved Sessions list](../images/kb/saz/saz-export-saved-sessions.png)
+
+- Use the **Import** option from the **Sessions** list's context menu and locate the *.saz file from the OS file system.   
+
+    ![Import SAZ into the **Sesssions** list](../images/kb/saz/saz-import-saved-sessions.png)
+
+All session snapshots saved in the local storage are also available as SAZ files in the `.fiddler` folder. You can manually access and backup your locally saved from the following paths:
+
+* Windows path
+    ```curl
+    C:\Users\<currently-logged-user>\.fiddler\<unique-fiddler-user-GUID>\Snapshots
```

Comment:
```suggestion
    %userprofile%\.fiddler\<unique-fiddler-user-GUID>\Snapshots
```
alternatively, to avoid confusion if windows is not installed in c: and to be consistent with the other OS paths (~)